### PR TITLE
New menu system, second iteration.

### PR DESCRIPTION
**Pull request by rebeccamadsen**
This is the next step for #245. This updates the navigation system to use a progress bar as a visual indicator throughout an interview, adds animation for stage changes, and fixes the "flashing" to the first prompt on a stage change.

Note that this does not include new menus for stages and settings, which will be iteration three, but it does remove the buttons. Instead, click on the timeline to open the stages menu. Settings menu is the last item in the stages menu.

### CODE REVIEW

**Comment by wwqrd** [comment]
This is looking fantastic!

File: src/containers/App.js [hunk 1]
```diff
@@ -25,8 +26,8 @@ const App = props => (
   })}
   >
     <div className="electron-titlebar" />
-    <SettingsMenu hideButton={props.isMenuOpen} />
-    <StageMenu hideButton={props.isMenuOpen} />
+    <SettingsMenu hideButton />
```
Comment (wwqrd) [inline src/containers/App.js]: Is this attribute redundant?

Comment (rebeccamadsen) [reply src/containers/App.js]: Yes, this was just trying to get the menu back in with minimal changes. But it wouldn't be tricky to clean that up a bit.

**Comment by bryfox** [comment]
I haven't reviewed code (it looks like wwqrd did), but this is working well for me.

**Pull request merged**